feature-utils/poly-look/test/react-components/contexts/mocks/pod-mock.js

```diff
@@ -0,0 +1,40 @@
+export const MOCKED_POD_RUNTIME = "podjs-mock";
+export const MOCKED_POD_RUNTIME_VERSION = "podjs-mock-version";
+export const mockFiles = [{ id: "mockFilename1" }, { id: "mockFilename2" }];
+
+const mockStorage = {
+  mockFilename1: "mockFilename1",
+  mockFilename2: "mockFilename2",
+};
+
+class MockerPodInfo {
+  async getRuntime() {
+    return MOCKED_POD_RUNTIME;
+  }
+
+  async getVersion() {
+    return MOCKED_POD_RUNTIME_VERSION;
+  }
+}
+
+export class MockerPod {
```

Comment:
Have you considered using `DefaultPod` for this? It does not really have any other functionality... At any rate, this looks like a common enough thing to put it in the `feature-api` space. Does not seem to have anything related to React anyway, so another option would be to just put it in the `test-utils` directory.

Reply:
Also, we are going to reactivate this ticket https://jira.polypoly.eu/browse/PROD4POD-807 We'd welcome any feedback.

Reply:
Given how short this is, I'd say it's fine for now, but we probably want to rethink this later.

Reply:
Given how short this is, I'd say it's fine for now, but we probably want to rethink this later.